programs/server/Server.cpp

```diff
@@ -891,6 +857,19 @@ if (ThreadFuzzer::instance().isEffective())
 
             global_context->updateStorageConfiguration(*config);
             global_context->updateInterserverCredentials(*config);
+
+#if USE_SSL && USE_INTERNAL_SSL_LIBRARY
+            try
+            {
+                CompressionCodecEncrypted::Configuration::instance().load(*config, "encryption_codecs");
+            }
+            catch (Exception & e)
+            {
+                std::cout << e.message() << std::endl;
+            }
+#else
+            LOG_WARNING(log, "Server was built without Base64 or SSL support. Encryption is disabled.");
+#endif
```

Comment:
I don't think we want `Server.cpp` to become a very long file. Let's make function `bool Configuration::tryLoad()` and move all that code in there. And please replace `std::cout` with `tryLogCurrentException`.